buildSrc/src/main/kotlin/publish-conventions.gradle.kts

```diff
@@ -0,0 +1,42 @@
+plugins {
+    `maven-publish`
+    signing
+}
+
+// empty xxx-javadoc.jar
+val javadocJar = tasks.register<Jar>("javadocJar") {
+    archiveClassifier.set("javadoc")
+}
+
+publishing {
+    repositories {
+        configureMavenPublication(this, project)
```

Comment:
The `configureMavenPublication()` util function is only used here - I think it'd be easier to copy-paste the function body here, and delete the util function.

WDYT?

Reply:
copy-pasted